src/inmanta/parser/plyInmantaLex.py

```diff
@@ -60,9 +60,22 @@
 reserved = {k: k.upper() for k in keyworldlist}
 
 # List of token names.   This is always required
-tokens = ["INT", "FLOAT", "ID", "CID", "SEP", "STRING", "MLS", "CMP_OP", "REGEX", "REL", "PEQ", "RSTRING", "FSTRING"] + sorted(
-    list(reserved.values())
-)
+tokens = [
+    "INT",
+    "FLOAT",
+    "ID",
+    "CID",
+    "SEP",
+    "STRING",
+    "MLS",
+    "CMP_OP",
+    "REGEX",
+    "REL",
+    "PEQ",
+    "RSTRING",
+    "FSTRING",
+    "ARITHMETIC_OP",
```

Comment:
I think a generic arithmetic token will not work. The arithmetic operators differ from e.g. the comparison operators in that they have unequal precedence (which honestly might have been a good idea for the comparison operators as well). Since precedence is specified by token, I think we need different tokens. At least one per precedence level, but honestly I don't see a strong reason to not just create one token per operator. You can always bundle them in a single parser production (rather than a lexer token).

It's been a while since I've been involved in the parser, so let me know if you think I'm missing something.